ios-base.xcodeproj/project.pbxproj

```diff
@@ -66,12 +68,18 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXFileReference section */
+		0386860B696BBA3A08D7F16C /* Pods-AcceptanceTests.staging.xcconfig */ = {isa = PBXFileReference; includeInIndex = 1; lastKnownFileType = text.xcconfig; name = "Pods-AcceptanceTests.staging.xcconfig"; path = "Target Support Files/Pods-AcceptanceTests/Pods-AcceptanceTests.staging.xcconfig"; sourceTree = "<group>"; };
```

Comment:
Please remove all pod related stuff before committing.

Reply:
Why is this an issue? Do we ignore this changes for some specific reason?

Reply:
We are leaving the base as clean as possible. The only file needed to install pods is the Podfile, all other autogenerated code changes with different cocoapod versions. Also,  it could cause conflicts with the init script, not sure though. You can clean with `pod deintegrate`.

Reply:
Okay, makes sense 👍